content/blog/easily-create-and-manage-aws-eks-kubernetes-clusters-with-pulumi/index.md

```diff
@@ -15,7 +15,7 @@ code to the rescue! We can now provision an entire EKS cluster with a
 CLI single gesture, thanks to [the `@pulumi/eks`
 package](https://github.com/pulumi/pulumi-eks). Let's see how.
 
-![Pulumi making EKS Easy](./easy-eks.png)
+{{< img src="easy-eks.png" alt="Pulumi making EKS Easy" >}}
```

Comment:
@chrsmith, @cnunciato,

I had to introduce this `img` shortcode because these images fall above `<!--more-->` and are therefore included in summary pages such as:

- /blog/tag/aws/page/2/easy-eks.png
- /blog/tag/kubernetes/page/2/easy-eks.png
- /blog/tag/kubernetes/page/2/status-rich.gif
- /blog/author/luke-hoban/easy-eks.png
- /blog/page/4/easy-eks.png
- /blog/page/7/status-rich.gif
- /blog/author/alex-clemmer/status-rich.gif

Since the images are relative to the content page, they end up being broken links in these summary pages.

If we want to keep these images with the page (vs. including them in `/static/images`), we'll need to use this shortcode so we can get a `RelPermalink` via `$.Page.Resources.GetMatch`.

The downside is this introduces some inconsistency in how images are handled in blog posts: above `<!--more-->` the shortcode _must_ be used, but below it regular markdown syntax or normal `<img>` HTML tags can be used. I can see this being a source of confusion for folks. But couldn't really think of a better solution (other than the `/static/images` approach, which we could use for these two cases). Do you have a preference?

If we're concerned about the inconsistency, we could use the shortcode for _all_ images in blog content (above and below `<!--more-->`). Or move to an approach where we move all images under `/static/images`. But since this has only come up in 2 posts among 80, it probably won't happen very often. Maybe just something to add to the blog guidance in #1370?

Reply:
Just lurking and trying to educate myself -- it does seem a bit confusing especially if you aren't very familiar with Hugo shortcodes. :+1: on adding this to the blog guidance.

Reply:
@cnunciato, any thoughts or alternatives?

Reply:
Hm, yeah. 🤔  

I'd probably argue for just moving the `<!--more-->`s to change the content of the summaries, then add a note to BLOGGING.md to not put images in summaries.

Reply:
![image](https://user-images.githubusercontent.com/274700/61495508-0d87a180-a96e-11e9-8956-3b029bb4936e.png)

![image](https://user-images.githubusercontent.com/274700/61495571-3740c880-a96e-11e9-99da-eb764c1af584.png)

Reply:
Yeah -- this looks all right to me, actually.

Reply:
Much simpler! Thanks, @cnunciato!